### ISSUE TITLE
Angularjs articles in Angular2+ filter

### ISSUE THREAD

**Issue by jeremyodell** [joeeames/ngDocIO_Issues#6]
Angular (2+) selected for filter/tab.

I am seeing at least 2 angularjs articles.

- BINDING TO DIRECTIVE CONTROLLERS IN ANGULAR 1.3

- EXPLORING ANGULAR 1.5: LIFECYCLE HOOKS

**Comment by TheKeithStewart** [comment]
Hi @jeremyodell! Thanks for pointing these out!  We are actively working on cleaning up some of the data on the site right now.

It looks like the two articles you mentioned have been updated to the correct version of Angular now.

**Comment by TheKeithStewart** [comment]
I just took another look at this and it appears that it is indeed still an issue.  

To reproduce, on the main page, if you do not enter a value in the "Search..." box and you choose the "Angular (2+)" filter button currently these two articles are displayed in the list of articles on the page.

**Comment by joeeames** [comment]
ok, so here's what's going on. As far as the system is concerned, this isn't actually "filtered" isntead it's showing like the most recent 50 articles to be submitted.

Perhaps we should make the angualr 2 button unselected? but then how does it get unselected once it's selected? it's a toggle button. 

Maybe there should be no "unfiltered" view. just default to only angular 2 articles....

**Comment by jeremyodell** [comment]
Just an idea here, you could make angularjs and angular checkboxes.  By default on page load angular is selected?

**Comment by joeeames** [comment]
checkboxes would then allow someone to select both v1 and v2 at the same time.....

**Comment by TheKeithStewart** [comment]
I like the checkboxes idea.  I would bet that it will not be a common thing to want to look for articles from both v1 and v2 at the same time but there could be a use for it.  For instance, there were some articles that were written about component architecture that were about v1 and the concepts covered would most like be perfectly applicable to v2 as well.

**Comment by joeeames** [comment]
yeah, migration articles could cover both versions. But the data architecture is so far away from supporting that concept.....

**Comment by joeeames** [comment]
so that change will take a while. I can either leave it as is, or make it so that it's always at least filtered by version, and there's never an "unfiltered" state. That's a pretty quick change. Should I do that for now?

**Comment by TheKeithStewart** [comment]
@joeeames, I think that the quick change that you have described sounds good for now.  Then, the data that is displayed on the right will accurately reflect the filter options that are chosen on the left.
